typedb/TypeDBSimulation.kt

```diff
@@ -59,9 +59,11 @@ abstract class TypeDBSimulation<out CONTEXT: Context<*, *>> protected constructo
         nativeClient.session(context.dbName, SCHEMA).use { session ->
             LOGGER.info("TypeDB initialisation of $name simulation schema started ...")
             val start = Instant.now()
-            val schemaQuery = Files.readString(schemaFile.toPath())
             session.transaction(WRITE).use { tx ->
-                tx.query().define(TypeQL.parseQuery(schemaQuery))
+                schemaFiles.forEach { schemaFile ->
```

Comment:
We should add a check to confirm the list is non-empty.

Reply:
I'm not sure this is necessary? I mean I can but not really sure what it adds. It's up to the maker of each simulation that depends on this repo to ensure it works correctly. Unlike the list of agents run, the schema files are hardcoded into each simulation.

Reply:
It seems likely that if an empty list of schema files is provided, it was a mistake. If you can think of a plausible case where this would be useful, then let's leave it as is.

Reply:
Yeah fair enough. Will do that now.